src/js/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -0,0 +1,136 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash/fp';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import SaveFormLink from '../save-in-progress/SaveFormLink';
+import SaveStatus from '../save-in-progress/SaveStatus';
+
+import { saveAndRedirectToReturnUrl, autoSaveForm, saveErrors } from './actions';
+import { toggleLoginModal } from '../../../login/actions';
+
+import { ReviewPage } from '../review/ReviewPage';
+import { setData, setPrivacyAgreement, setEditMode, setSubmission, submitForm, uploadFile } from '../actions';
+
+class RoutedSavableReviewPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.autoSave = this.autoSave.bind(this);
+    this.debouncedAutoSave = _.debounce(1000, this.autoSave);
```

Comment:
For consistency, can we make `autoSave` an arrow function? Not sure if that'd mess up `debouncedAutoSave` or not.